docs/samples/tasks.yaml

```diff
@@ -109,6 +109,7 @@
   # parameter must be provided.
   query_type: !FixedSize
     max_batch_size: 100
+    max_batch_duration: null
```

Comment:
nit: could we allow this to default to `None` if unspecified?

Reply:
Yes, the last case in the `task::tests::query_type_serde` test checks that omitting it is supported. I just added this to the sample file for the sake of illustration.